Foodiea/Manager/APIManager.h

```diff
@@ -0,0 +1,22 @@
+//
+//  APIManager.h
+//  Foodiea
+//
+//  Created by Laura Jankowski on 7/8/22.
+//
+
+#import <Foundation/Foundation.h>
+#import <Parse/Parse.h>
+NS_ASSUME_NONNULL_BEGIN
+
+@interface APIManager : NSObject
```

Comment:
Let's change the name of this. This manager is for parse so the name should resemble that so it identifies it versus other managers. What will you do once you start working with yelp.